.github/workflows/linux.yml

```diff
@@ -97,12 +101,17 @@ jobs:
           make -C build -j$(nproc)
           make -C build -j$(nproc) utrecht_tiles
           make -C build -j$(nproc) tests gurka
-      
-      - name: Save cache
-        uses: actions/cache/save@v4
+
+      - name: Save ccache
+        if: always()
+        uses: tespkg/actions-cache/save@v1
         with:
+          endpoint: hel1.your-objectstorage.com
+          accessKey: ${{ secrets.HETZNER_S3_ACCESS_KEY }}
+          secretKey: ${{ secrets.HETZNER_S3_ACCESS_SECRET }}
+          bucket: gha-cache
           path: /home/runner/.cache/ccache
-          key: ccache-release-linux-x86_64-${{ env.BRANCH_NAME }}-${{ env.TIMESTAMP }}
```

Comment:
at least with this new action we don't need the timestamp anymore. it'll overwrite the cache by default, which is perfect, we don't need more than one cache per branch.